src/App.js

```diff
@@ -1,5 +1,18 @@
+import { BrowserRouter, Routes, Route } from "react-router-dom";
+import Admin from "./pages/Admin/Admin";
+import Employee from "./pages/Employee/Employee";
+import Signin from "./pages/Signin/Signin";
+
 function App() {
-  return <></>;
+  return (
+    <BrowserRouter>
+      <Routes>
+        <Route exact path="/employee" element={<Employee></Employee>}></Route>
+        <Route exact path="/admin" element={<Admin></Admin>}></Route>
+        <Route exact path="/" element={<Signin></Signin>}></Route>
```

Comment:
Is this prop `element` actually existing?

Reply:
I am using react-router-dom v6. In this version, many props and components had been renamed. And in this case, component (v5) has been changed to element

Reply:
ok